src/app/file-manager/transferables-grid/transferables-grid.component.ts

```diff
@@ -28,10 +28,12 @@ import { WarningModalComponent } from '../warning-modal/warning-modal.component'
 
 export class TransferablesGridComponent implements OnInit, AfterViewInit {
   static isExporting: Boolean = false;
+  static firstIteration: Boolean = true;
```

Comment:
The reason these are not local variables, is because their values are not persisting through the app execution.

When receiving info from observables we get a new instance from the component, which redefines all these variables, needed to reflect changes in the Ui. 

We were discussing a possible solution that is implement Redux store in order to reflect the app state allowing us to remove every static variable, but at the moment this would be a temporal solution.

Reply:
It's OK to leave these as statics for now. Maybe add a comment about how a new component is created each time but this state has a longer lifetime than the component. The goal would be to put the properties on an object that has the correct lifetime, maybe the Store is the best place.